### PR TITLE
Fix pd -N issue, with correct offset for bw-disassembling

### PR DESCRIPTION
**Pull request by m1ghtym0**

### CODE REVIEW

**Comment by m1ghtym0** [comment]
Created PR in radare2-regressions:
https://github.com/radare/radare2-regressions/pull/590

File: libr/core/cmd_print.c [hunk 1]
```diff
@@ -2974,7 +2974,7 @@ static int cmd_print(void *data, const char *input) {
 						memcpy (block, core->block, bs);
 						r_core_read_at (core, addr+bs, block+bs, instr_len-bs); //core->blocksize);
 						core->num->value = r_core_print_disasm (core->print,
-								core, addr, block, instr_len, l, 0, 1);
+								core, core->offset, block, instr_len, l, 0, 1);
```
Comment (radare) [inline libr/core/cmd_print.c]: this seems inconsistent or confusing to me, the prev line is doing read from addr+bs at block+bs, and here you are refering core->offset as the base address of the block buffer.

**Comment by radare** [comment]
there are some pd-N BROKEN tests, is this PR fixing them?

**Comment by radare** [comment]
oh i see the logic, addr is modified by bwlen() so keeping that core->offset as base for block is good, but the whole thing should be cleaned up to be more readable. merging

**Pull request merged**